src/Simplex/FileTransfer/Server.hs

```diff
@@ -248,6 +252,9 @@ processXFTPRequest HTTP2Body {bodyPart} = \case
         withFileLog $ \sl -> do
           logAddFile sl sId file ts
           logAddRecipients sl sId rcps
+        stats <- asks serverStats
+        atomically $ modifyTVar' (filesCreated stats) (+ 1)
+        atomically $ modifyTVar' (fileRecipients stats) (+ length rks)
```

Comment:
why are these and other stats updates in separate atomically?

Reply:
same as in SMP server really, there is no much benefit in them being strongly consistent, but it reduces the deadlock risks.